### PR TITLE
kinetis renaming

### PR DESCRIPTION
**Pull request by perigoso**
Just a general cleanup and function renaming, separated from the #228 fix attempt branch

Built atop #1094

### CODE REVIEW

**Review by dragonmux — APPROVED** [review]
LGTM, we'll merge this once our PR has been reviewed and merged by Esden.

**Comment by esden** [comment]
I merged #1094, can you rebase so I can re-review and merge this?

**Comment by esden** [comment]
Thank you! Merged with 0c18fcefe3a5e1f40bbf746310e9d36119e6610d

**Pull request closed without merging**
